pages/integrations/output_framework.rst

```diff
@@ -0,0 +1,142 @@
+  outgoing payload and store it in the ``pipeline_output`` field of the message.  This can 
+  be accomplished in the pipeline by using the ``set_field`` :doc:`built-in function<../pipelines/functions>`.
+
+**Output Transports**
+
+- **Enterprise STDOUT**: Formatted messages will be displayed on the system's console. 
+  This is included primarily as a debugging tool for pipeline changes.
+- **Enterprise TCP Raw/Plaintext**: Formatted messages will be sent as UTF-8 encoded plain 
+  text to the configured TCP endpoint (IP address and port).
+- **Enterprise TCP Syslog**: Formatted messages will be sent as the ``MSG`` portion of a 
+  standard Syslog message per section 6.4 of the `Syslog specification <https://tools.ietf.org/html/rfc5424>`_.  
+  The Syslog message will be sent to the configured TCP endpoint (IP address and port).
+
+
+Output Framework Options
+------------------------
+
+The Enterprise Output Framework is capable of processing messages at very high throughput 
+rates. Many hardware factors will affect throughput (such as CPU clock speed, number of 
+CPU cores, available memory, and network bandwidth). Several Output Framework configuration 
+options are available to help you tune performance for your throughput requirements and 
+environment.
+
+Common Options
+^^^^^^^^^^^^^^^
+
+.. image:: /images/integrations/output-framework-config-01.png
+.. image:: /images/integrations/output-framework-config-02.png
+
+- **Title**: The name of the Output
+- **Send Buffer Size**: The size of the sender ring buffer
+- **Concurrent message processing pipelines**: The number of pipeline instances that will 
+  be allowed to run at any given time.  If this is set to 0, pipeline execution will be
+  skipped even if a pipeline is selected from the Pipeline dropdown.
+- **Concurrent output payload formatters**: The number of formatter instances that will be
+  allowed to run at any given time.  If this is set to 0, the Output will fail.
+- **Concurrent message senders**: The number of sender instances that will be allowed to
+  run at any given time.  If this is set to 0, the Output will fail.
+- **Journal Segment Size**: The soft maximum for the size of a journal segment file
+- **Journal Segment Age**: The maximum amount of time journal segments will be retained if
+  there is storage to do so
+- **Maximum Journal Size**: The maximum size of the journal
+- **Maximum Journal Message Age**: The maximum time that a message will be stored in the
+  disk journal
+- **Journal Buffer Size**: The size of the memory buffer for messages waiting to be written 
+  to the journal. This must be a power of two.
+- **Journal Buffer Encoders**: The number of concurrent encoders for messages being written
+  to the journal.
+- **Output Processing Pipeline**: The pipeline which will process all messages sent to the
+  Output
+- **Outbound Payload Format**: The format that will be used for outgoing message payloads
+
+
```

Comment:
Do we have any notes, on where the Journal will be stored at by default?  Just for documentation sake so they know where to look at.

Reply:
I believe all on-disk journal data is stored in the same location specified by the `message_journal_dir` value in the `graylog.conf` file.